### ISSUE TITLE
[BUG] Filtering by click on the correspond does not work

### ISSUE THREAD

**Issue by marvinwankersteen** [astubenbord/paperless-mobile#184]
**Describe the bug**
It's no longer possible to filter documents by click/tab on the correspond in the main view.
Instead it's possible by click/tab on the tags.

**To Reproduce**
1. Go to 'Documents'
2. Click on the choosen correspond. E.g. "Mr. X"
3. Nothing happens

**Expected behavior**
Documents should be filtered by the correspond "Mr. X".

**Device Information:**
 - Device: OnePlus 5, Samsung Galaxy Tab S5e
 - OS: LOS20 with microG, LOS20 crDroid
 - Paperless Mobile version: 2.3.2+2037
 - Paperless Server version: 1.14.5

In the previous version 2.1.0 it worked.

PS: I love this app 👍

**Comment by astubenbord** [comment]
Yeah, sorry I took that feature out because the hitboxes of both correspondent and document type are too small to intuitively consider them as an element of interaction. Is that something you use on a regular basis?


PS: Thanks for the compliment :)

**Comment by marvinwankersteen** [comment]
Ah, okay.
I had already noticed that you have to press very precisely. However, I got used to it and had no more problems.

I actually used it a lot, because I find the manual search for the correspondent more cumbersome until you have selected or found the right one in the list.

As an alternative, would it be possible to set the filter when you have opened a document? The tags and the correspondent are also displayed there.

**Comment by astubenbord** [comment]
The mechanisms/callbacks to achieve this behavior are still present in the current codebase and it wouldn't require much effort to implement in both ways (click directly on the label in on the list tile and on the opened document).I'll convert this to a feature request.
